gnovm/tests/files/zrealm_crossrealm22a1.gno

```diff
@@ -0,0 +1,24 @@
+// PKGPATH: gno.land/r/crossrealm_test
+package crossrealm_test
+
+import (
+	"std"
+
+	"gno.land/r/demo/tests/crossrealm"
+	"gno.land/r/demo/tests/crossrealm_b"
+)
+
+type Foo struct{}
+
+func main() {
+	f := crossrealm_b.Fooer
+	crossrealm.SetFooerGetter(func() crossrealm.Fooer {
+		f1 := Foo{}
```

Comment:
`f1` is owned by `main`, then by package. so the funcLit is not permitted to be attached to external realm.